src/main/java/com/uptech/accounted/controller/TransactionController.java

```diff
@@ -227,10 +231,16 @@ public void initialize(URL location, ResourceBundle resources) {
     loadRecipients();
     transactionTable.setOnMouseClicked(event -> {
       Transaction selectedItem = transactionTable.getSelectionModel().getSelectedItem();
-      cbInitiator.getSelectionModel().select(selectedItem.getInitiator().getCode() + "-" + selectedItem.getInitiator().getName());
-      cbDepartment.getSelectionModel().select(selectedItem.getDepartment().getCode() + "-" + selectedItem.getDepartment().getName());
-      cbRecipient.getSelectionModel().select(selectedItem.getRecipient().getCode() + "-" + selectedItem.getRecipient().getName());
-      cbLedgerType.getSelectionModel().select(selectedItem.getLedgerType().getLedgerCode() + "-" + selectedItem.getLedgerType().getLedgerName());
+      cbInitiator.getSelectionModel()
+          .select(selectedItem.getInitiator().getCode() + "-" + selectedItem.getInitiator().getName());
+      cbDepartment.getSelectionModel()
+          .select(selectedItem.getDepartment().getCode() + "-" + selectedItem.getDepartment().getName());
+      cbRecipient.getSelectionModel()
+          .select(selectedItem.getRecipient().getCode() + "-" + selectedItem.getRecipient().getName());
```

Comment:
Subledger dropdown in Transaction Form will have the format "LedgerCode-SubledgerCode-SubledgerName". So that two subledgers having same names and codes but with different ledgers don't mix up